tools/maven/checkstyle.xml

```diff
@@ -180,6 +180,11 @@ This file is based on the checkstyle file of Apache Beam.
 			<property name="severity" value="error"/>
 		</module>
 
+		<module name="IllegalImport">
+			<property name="illegalPkgs"
+					  value="org.mockito, org.powermock"/>
+			<message key="import.illegal" value="{0}. Please refer to the coding guidelines."/>
```

Comment:
It can be explained more clearly here like `we should avoid using mocks, please refer to the coding guidelines[https://flink.apache.org/how-to-contribute/code-style-and-quality-common/#avoid-mockito---use-reusable-test-implementations]` or other better descriptions.